content/references/rippled-api/api-conventions/currency-formats.md

```diff
@@ -33,9 +33,16 @@ Issued currencies in the XRP Ledger are represented with a custom format with th
 
 Internally, `rippled` represents numbers for issued currencies in a custom number format. This format can store a wide variety of assets, including those typically measured in very small or very large denominations. Unlike typical floating-point representations of non-whole numbers, this format uses integer math for all calculations, so it always maintains 15 decimal digits of precision. Unlike "arbitrary precision" number formats, the custom format can always be stored in a fixed size of 64 bits.
```

Comment:
What does "internally" mean? Does it mean that as a dev using the `rippled` API, you'll never encounter values in this format in the API's requests and responses, but we're telling you how `rippled` stores these values behind the scenes to show how it supports high precision?

Reply:
The idea behind this paragraph is:

- To let people know how to match `rippled`'s math exactly when it comes to issued currency amounts, so they know what to expect or can predict the outcomes of transactions down to the lowest decimal place. Or, more likely, to convince them that it's not worth the effort because they'd have to implement a bunch of low-level number routines, but they can come reasonably close by using a "big decimal" library with 15 digits of precision.
- To inform those implementing transaction serialization/signing that the binary format fits all that stuff in 64 bits, which also serves as a justification for why the first point is the way it is.

Some of this stuff is a little extra low-level and should probably get migrated to the Serialization Format page eventually (sometime after #483 is merged).

Reply:
Thank you for taking the time to provide such thorough and thoughtful context for me. Got it. 🙌